plugins/flatpak/gs-plugin-flatpak.c

```diff
@@ -992,13 +989,11 @@ gs_plugin_flatpak_update (GsPlugin *plugin,
 	}
 	for (guint i = 0; i < gs_app_list_length (list_tmp); i++) {
 		GsApp *app = gs_app_list_index (list_tmp, i);
-		g_autofree gchar *ref = NULL;
 
-		ref = gs_flatpak_app_get_ref_display (app);
```

Comment:
Andre said this change is needed because the proxy app is used in this code path, but the question remains of whether we should instead skip the call to `gs_flatpak_refine_app()`

Reply:
I think we should still call gs_flatpak_refine_app on the proxy app given the eos plugin has some specific code to refine proxy apps (although I am not 100% sure it is reached by this call here).

Reply:
Ah looks like `gs_flatpak_refine_app_unlocked()` is a no-op if the app is a proxy, so never mind.